src/ts/waterfall/sub-components/svg-marks.ts

```diff
@@ -32,9 +32,10 @@ export function createMarks(context: Context, marks: Mark[]) {
     });
 
     const lastMark = marks[i - 1];
-    if (lastMark && mark.x - lastMark.x < 1) {
-      lineLabel.setAttribute("x", lastMark.x + 1 + "%");
-      mark.x = lastMark.x + 1;
+    const minDistance = 2.5; // minimum distance between marks
+    if (lastMark && mark.x - lastMark.x < minDistance) {
```

Comment:
will mark.x always be greater then lastMark.x or should this comparision use absolutes instead?